app/customer_routes.py

```diff
@@ -0,0 +1,104 @@
+from flask import Blueprint, jsonify, make_response, request
+
+from app import db
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
+from app.rental_routes import deletes_rentals
+
+customers_bp = Blueprint("customers",__name__, url_prefix="/customers")
+
+@customers_bp.route("",methods=["GET"])
+
+def get_customers():
+    customers = Customer.query.all()
+    customers_response = []
+    for customer in customers:
+        customers_response.append(customer.to_dict())
+
+    return jsonify(customers_response),200
+
+@customers_bp.route("/<customer_id>",methods=["GET"])
+def get_customer_by_id(customer_id):
+    
+    try:
+        customer_id = int(customer_id)
+    except ValueError:
+        return make_response(jsonify({"error": "Invalid ID"}), 400)
+    customer = Customer.query.get(customer_id)
+    if not customer:
+        return make_response(jsonify({'message': f'Customer {customer_id} was not found'}),404)
+    return customer.to_dict(),200
+
+@customers_bp.route("/<cust_id>/rentals",methods=["GET"])
+def get_customer_rentals(cust_id):
+
+    customer = Customer.query.get(cust_id)
+    if not customer:
+        return make_response(jsonify({'message': f'Customer {cust_id} was not found'}),404)
+
+    all_rentals = []
+    rentals = db.session.query(Rental).filter(Rental.customer_id==cust_id).all()
+    for rental in rentals:
+        video = Video.query.get(rental.video_id)
+        all_rentals.append({"title":video.title})
+    return make_response(jsonify(all_rentals),200)
```

Comment:
I think you can also use the relationships for this.

`customer.rentals # gives collection of rentals for this customer`